src/frontend/citizen-portal/src/assets/app.config.json

```diff
@@ -1,7 +1,7 @@
 {
   "production": false,
   "environment": "dev",
-  "version": "2.0.0",
+  "version": "2.0.2",
```

Comment:
I think this is now 0.0.0 in main to be clear that version numbers come from helm, not the code base.

Reply:
I will change this